app/src/main/java/org/mozilla/focus/MainActivity.kt

```diff
@@ -142,15 +142,6 @@ class MainActivity : LocaleAwareAppCompatActivity(), BrowserFragmentCallbacks, U
         }
     }
 
-    override fun dispatchKeyEvent(event: KeyEvent): Boolean {
-        val browserFragment = supportFragmentManager.getBrowserFragment()
-        return if (browserFragment != null && browserFragment.isVisible) {
-            browserFragment.dispatchKeyEvent(event) || super.dispatchKeyEvent(event)
-        } else {
-            super.dispatchKeyEvent(event)
-        }
-    }
```

Comment:
I think code removal is a good candidate for being broken into a separate PR, since nothing relies on it.